src/styles/typography.scss

```diff
@@ -28,14 +28,6 @@ h1, h2, h3, h4, h5, h6, p, ul {
 
 // Titles ///////////////////////////
 
-.title-hero {
-  font-weight: var(--text-weight-black);
-  font-size: var(--space-5);
-  line-height: 1.1;
-
-  @include media(">=notebook") { font-size: var(--space-8); }
-}
-
```

Comment:
Don't know if I like removing it from here, it's useful for future customization to see all the major typography sizes on the same page.

Reply:
Hm! That is something to consider.
So, just let's keep it here? 👀

Reply:
We decided together to keep this on typography and adapt in hero. 🎉